app-typescript/components/MultiSelect.tsx

```diff
@@ -46,55 +38,60 @@ export class MultiSelect<T> extends React.Component<IProps<T>, IState<T>> {
     constructor(props: IProps<T>) {
         super(props);
         this.state = {
-            value: [],
+            value: this.props.value ? this.props.value : [],
```

Comment:
Make it explicit with either `this.props.value != null ? this.props.value : []` or `this.props.value ?? []`